cmd/influx_inspect/dumptsi/dumptsi.go

```diff
@@ -244,6 +244,9 @@ func (cmd *Command) printSeries(sfile *tsdb.SeriesFile) error {
 			break
 		}
 		name, tags := tsdb.ParseSeriesKey(sfile.SeriesKey(e.SeriesID))
+		if name == nil {
+			continue
+		}
```

Comment:
@jdockerty @davidby-influx Should we be checking for `len(name) == 0` instead of `name == nil`? Checking the `len` would also catch the case where the name is empty in addition to nil. Or is an empty name valid?

Reply:
For any reader's clarity, we discussed this offline and concluded that `len(name) == 0` was a safer/wider check to handle potential edge cases, whilst also still covering my initial intention.

I've altered this now 💯